README.md

```diff
@@ -193,6 +193,7 @@ Table of Contents
   * [IP.City](https://ip.city) — 100 free IP geolocation requests per day
   * [Abstract API](https://www.abstractapi.com) — API suite for a variety of use cases including IP geolocation, gender detection or even email validation.
   * [algorithmia.com](https://algorithmia.com/) — Host algorithms for free. Includes free monthly allowance for running algorithms. Now with CLI support.
+  * [Apache APISIX](https://apisix.apache.org/) - Dynamic, real-time, high-performance API Management Platform (API Gateway, Ingress Controller, including Service Mesh) by Apache Software Foundation.
```

Comment:
I do not believe this is a offering delivered As A Service?

Reply:
Just noted `This list is only for as-a-Service offerings, not for self-hosted software.`, I would recommend API7 Cloud, cc @tokers.